evenOccurrences/evenOccurrences.js

```diff
@@ -10,6 +10,16 @@
  *    * console.log(onlyEven); //  4
  *    */
 
+
+
 const evenOccurence = (arr) => {
   // Your code here.
-};
\ No newline at end of file
+  const onlyEven = evenOccurence([1, 7, 2, 4, 5, 1, 6, 8, 9, 6, 4, 1]);
```

Comment:
If you're doing this for testing purposes (i.e. running your code) you don't need to define this const inside your function body. In fact  you shouldn't. :)